test/features/voucher_activation.feature

```diff
@@ -0,0 +1,23 @@
+Feature: The VoucherActivation API is used to activate a pre-activated voucher
+  This API requests for voucher activation by sending the voucher serial number and Gov Stack BB
+
+  Request endpoint: PATCH /vouchers/voucher_activation
+
+  Background: The user wants to activate pre-activated voucher
+    Given The user wants to activate pre-activated voucher
+
+  Scenario: The user successfully activates a pre-activated voucher
+    When The user triggers an action with a valid payload to activate a pre-activated voucher
+    Then The user successfully activates a pre-activated voucher
+
+  Scenario: The user is not able to activate a pre-activated voucher because of an empty payload
+    When The user triggers an action with an empty payload to activate a pre-activated voucher
+    Then The result of the operation returns an error because of empty payload
+
+  Scenario: The user is not able to activate a pre-activated voucher because of an invalid voucher serial number
+    When The user triggers an action with an invalid serial number to activate a pre-activated voucher
+    Then The result of the operation returns an error because of an invalid serial number
+
+  Scenario: The user is not able to activate a pre-activated voucher because of an invalid Gov Stack Building Block
```

Comment:
Do you mean the user is not able to activate a pre-activated voucher because of Gov Stack Building Block does not exist? If yes, please change the title and error message.
<img width="679" alt="Zrzut ekranu 2023-01-9 o 12 51 46" src="https://user-images.githubusercontent.com/80507519/211301973-2f3c4a2d-caca-4ac8-8509-c77701c39df5.png">

Reply:
@mgrochalskisoldevelo

Reply:
@KarolinaKopacz yes, that's what i meant

https://github.com/GovStackWorkingGroup/bb-payments/pull/7/commits/823af1d2e95347ed8ffc773d87508f276b27fa84

![image](https://user-images.githubusercontent.com/119664031/212293039-25b6675b-39ee-4b50-aad8-c64d43af5587.png)

Reply:
I didn't see those changes, sorry :)